LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Smaranjit Ghose, Anush Bhatia
```

Comment:
Maybe change this copyright line?
Perhaps change year to `2021`, and copyright holder to `Tech-Matrix`